packages/dds/tree/src/feature-libraries/chunked-forest/chunkTree.ts

```diff
@@ -339,7 +345,15 @@ export interface ChunkPolicy {
 	shapeFromSchema(schema: TreeNodeSchemaIdentifier): ShapeInfo;
 }
 
-function newBasicChunkTree(cursor: ITreeCursorSynchronous, policy: ChunkPolicy): BasicChunk {
+export interface ChunkCompressor {
+	policy: ChunkPolicy;
+	idCompressor?: IIdCompressor;
+}
```

Comment:
I'm guessing you never reassign these fields, so they should probably be readonly.  Also a short doc comment on this type would be good.

Also a doc comment on the idCompressor at least covering what what changes if you provide it.

Reply:
On of the nice thigns about making a interface for this is you can put nice docs on the members without having to duplicate them to every function passing this data around. Lets use that :)

Reply:
Updated `ChunkCompressor` fields to readonly, and added doc comment for the id compressor.